app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    ls = [Person(name=person["name"], age=person["age"]) for person in people]
```

Comment:
Please, use more descriptive name for the variable, not `ls`

Reply:
list_data = [Person(name=person["name"], age=person["age"])
                 for person in people
                 ]

Reply:
list_data is not a good name also. Variable name should be self-desriptive, so you read it and understand what is in it

Reply:
Fix the variable name, please